constructorio-client/src/test/java/io/constructor/client/ConstructorIOBrowseItemsTest.java

```diff
@@ -303,4 +303,18 @@ public boolean test(FilterFacet f) {
         assertNotNull("browse facet [Brand] exists", brandFacet);
         assertTrue("browse facet [Brand] is hidden", brandFacet.getHidden());
     }
+
+    @Test
+    public void BrowseItemsShouldReturnAResultWithFilterMatchTypes() throws Exception {
+        ConstructorIO constructor = new ConstructorIO("", apiKey, true, null);
+        UserInfo userInfo = new UserInfo(3, "c62a-2a09-faie");
+        List<String> ids = Arrays.asList("10001", "10002");
+        BrowseItemsRequest request = new BrowseItemsRequest(ids);
+        request.getFilterMatchTypes().put("Color", "any");
+        BrowseResponse response = constructor.browseItems(request, userInfo);
+
+        assertTrue("browse result id exists", response.getResultId() != null);
+        assertTrue("request exists", response.getRequest() != null);
+        assertTrue("browse results exist", response.getResponse().getResults().size() >= 0);
+    }
```

Comment:
Can we check the request returned in the response, make sure filter_match_types is there?